js/latoken.js

```diff
@@ -0,0 +1,635 @@
+            let id = currency['currencyId'];
+            let symbol = currency['symbol'];
+            let name = currency['name'];
+            let precision = currency['precission'];
+            let type = currency['type'];
+            let fee = currency['fee'];
+            result.push ({
+                'id': id,
+                'symbol': symbol,
+                'name': name,
+                'precision': precision,
+                'type': type,
+                'fee': fee,
+            });
+        }
+        return result;
+    }
+
+    parseTrade (trade, market = undefined) {
+        let info = trade;
+        let type = undefined;
+        let fee = undefined;
+        return {
+            'info': info,
+            'timestamp': trade.timestamp,
+            'datetime': this.iso8601 (trade.timestamp),
+            'symbol': trade.symbol,
+            'pairId': trade.pairId,
+            'id': trade.id,
+            'order': trade.orderId,
+            'type': type,
+            'side': trade.side,
+            'price': trade.price,
+            'amount': trade.amount,
+            'cost': trade.price * trade.amount,
+            'commision': trade.commision,
+            'fee': fee,
+        };
+    }
+
+    async fetchTrades (symbol, limit = 100, params = {}) {
+        await this.loadMarkets ();
+        let market = this.market (symbol);
+        let resp = {
+            'symbol': market['symbol'],
+        };
+        let response = await this.publicGetMarketDataTrades (this.extend (resp, params));
+        let pairId = response.pairId;
+        let sym = response.symbol;
+        let tradeCount = response.tradeCount;
+        let result = [];
+        for (let i = 0; i < response.trades.length; i++) {
+            let resp = response.trades[i];
+            result.push ({
+                'pairId': pairId,
+                'symbol': sym,
+                'tradeCount': tradeCount,
+                'side': resp.side,
+                'price': resp.price,
+                'amount': resp.amount,
+                'timestamp': resp.timestamp,
+            });
+        }
+        let trades = this.parseTrades (result);
+        return trades;
+    }
+
+    async fetchMyTrades (symbol = undefined, params = {}, limit = 10) {
+        await this.loadMarkets ();
+        let request = {
+            'symbol': symbol,
+            'limit': 1000000000,
+        };
+        let response = await this.privateGetOrderTrades (this.extend (request, params));
+        let pairId = response.pairId;
+        let sym = response.symbol;
+        let tradeCount = response.tradeCount;
+        let result = [];
+        for (let i = 0; i < response.trades.length; i++) {
+            let resp = response.trades[i];
+            result.push ({
+                'pairId': pairId,
+                'symbol': sym,
+                'tradeCount': tradeCount,
+                'id': resp.id,
+                'orderId': resp.orderId,
```

Comment:
These lines and below are violating: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#accessing-dictionary-keys

Reply:
done